### PR TITLE
[SIMPLE_FORMS] feat: update pdf url and supporting logic

### PR DESCRIPTION
**Pull request by pennja**
## Summary

- *This work is behind a feature toggle (flipper): YES*
- *This PR includes updates to the response payload, S3 client logic, and temp file directory path. Specifically:*
  - *Updated the `pdf_url` key in the response payload to align with frontend requirements.*
  - *Enhanced S3 client logic to include the manifest, addressing dependencies on this data for accurate processing.*
  - *Refined the query path for the temporary file, ensuring the final directory includes the directory housing the file to streamline file lookup and upload operations.*
- *(Not a bug)*
- *These updates resolve issues in payload consistency, S3 logic completeness, and file upload stability, which are necessary to ensure accurate data flow from the backend to frontend components.*
- *Team: Veteran Facing Forms team, maintenance of this component is handled by our team.*
- *(No feature toggle introduced)*

## Related issue(s)

- *[Implement S3 PDF Storage and Pre-Signed URL Generation for Successful Submissions](https://github.com/department-of-veterans-affairs/VA.gov-team-forms/issues/1622)*
- *No previous change applicable to this code.*
- *No epic associated.*

## Testing done

- [x] *New code is covered by unit tests.*
- *Old behavior: Prior to the update, the payload lacked the `pdf_url` key, the manifest was missing in S3 logic, and the directory for temporary file uploads lacked precision, leading to potential file location issues.*
- *Verification steps:*
  - *Confirm that `pdf_url` is now included correctly in the response payload and received by the frontend as expected.*
  - *Ensure that the manifest is correctly passed through the S3 client logic without errors.*
  - *Validate that the updated directory path logic accurately locates the temp file for upload, with no errors during the query and upload process.*
- *No flipper involved, so flipper testing is not required.*

## What areas of the site does it impact?
*This impacts the backend API’s response handling, S3 client operations, and file management logic for temporary file uploads. No other areas are expected to be impacted.*

## Requested Feedback

*None required; changes are straightforward. However, if there are concerns regarding the manifest handling in S3 logic or if further testing is recommended, please advise.*

### CODE REVIEW

**Review by rhasselle-oddball — DISMISSED** [review]

**Review by rhasselle-oddball — APPROVED** [review]

File: lib/simple_forms_api/form_remediation/configuration/vff_config.rb [hunk 1]
```diff
@@ -7,7 +7,7 @@ module FormRemediation
     module Configuration
       class VffConfig < Base
         def s3_settings
-          Settings.vff_simple_forms.aws
```
Comment (rmtolmach) [inline lib/simple_forms_api/form_remediation/configuration/vff_config.rb:10]: @pennja Will this setting be used anywhere now? If not, you can delete it from the manifest repo. I see you added it in https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/pull/3116/ and https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/pull/3117

Comment (pennja) [reply lib/simple_forms_api/form_remediation/configuration/vff_config.rb]: Oof, great catch. This change and the changes to the uploader were never supposed to be added. Those were local changes for testing purposes. I've reverted those changes out. Thank you!

**Comment by pennja** [comment]
@rmtolmach I've updated the code to remove those files that were mentioned. This PR is ready for review again!

**Review by Thrillberg — APPROVED** [review]

**Review by rmtolmach — APPROVED** [review]

File: lib/simple_forms_api/form_remediation/configuration/vff_config.rb (hunk 1 above)
Comment (rmtolmach) [reply lib/simple_forms_api/form_remediation/configuration/vff_config.rb]: oh wow. I'm glad I said something!

**Pull request merged**